src/test/ledger/env.ts

```diff
@@ -114,6 +122,7 @@ export async function setupEnv(
 		const tx = await call(arg1, arg2, {
 			nonce: nonce++,
 		});
+		await mineBlocks(2); // one block containing TX + 1 confirmation.
```

Comment:
Why is it important that each token registration is in a separate block? Don't the `mineBlocks()` call after `addMinter` plus the final `sealEpoch` call suffice?

Reply:
Just wanted to keep the error handling which was already in place. I am not sure whether or not the additional error handling is superfluous or not. Maybe you can put the final nail in the coffin and give your opinion. Putting them all in a single block is possible, I just do not think that the error output (in case of an error) is easily digestible since ganache errors are already ugly on their own :see_no_evil:

Reply:
OK, if it helps debugging ganache output, then I'm fine with that ;) But wouldn't it still suffice to mine a single block? Because now you're mining one block containing two txs, and then another one containing zero txs.

Reply:
You are correct, I was under the impression that `tx.wait(numOfConfirmations = 1)` meant that one **more** block had to be mined for `tx.wait(1)` to return (and not wait `blockTime` seconds).

But this is not the case, so we can remove the `2`. Thanks.